ownCloud/Client/Viewer/DisplayViewController.swift

```diff
@@ -364,15 +345,57 @@ class DisplayViewController: UIViewController, Themeable {
 		}
 	}
 
+	// MARK: - Update control
+	enum UpdateStrategy {
+		case ask
+		case alwaysUpdate
+		case neverUpdate
+	}
+	var updateStrategy : UpdateStrategy = .alwaysUpdate
+
+	func shouldRenderItem(item: OCItem, isUpdate: Bool, shouldRender: @escaping (Bool) -> Void) {
+		if isUpdate {
+			switch updateStrategy {
+				case .ask:
+					OnMainThread {
+						let alert = UIAlertController(title: NSString(format: "%@ was updated".localized as NSString, item.name ?? "File".localized) as String, message: "Would you like to view the updated version?".localized, preferredStyle: .alert)
```

Comment:
`File" is missing in localized strings table

Reply:
Good catch! Added that. Thanks.